Automattic/jetpack
# Fix double fetch, move search into state

## Proposed changes:
This PR moves the search term into the state and fixes the double fetch seen on searches and paging

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Add these lines on contact-form.php

```php
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

Then access the new Form responses inbox at `/wp-admin/admin.php?page=jetpack-forms`

Use the searchbox and paging in any combination, see that it works as expected. Keep an eye on the network tab on devtools, see that on any action, only a single request is sent.
